packages/venia-concept/src/ServiceWorker/Utilities/messageHandler.js

```diff
@@ -109,7 +109,11 @@ export const sendMessageToClient = (client, type, payload) =>
             }
         };
 
-        client.postMessage({ type, payload }, [channel.port2]);
+        if (client) {
+            client.postMessage({ type, payload }, [channel.port2]);
```

Comment:
Could we do something like `const portToClient = channel.port2`  and then just use `portToClient` from there on out?

Not a huge deal, I'm just thinking this is an easy place for bugs to creep in.

Reply:
We only use `channel.port2` to send it over a message. If we have to rename it, it will only be for a single-use. Creating a variable for a single-use is usually discouraged.